### DISCUSSION TITLE
Style outer radio button

### DISCUSSION THREAD

**Discussion by muinmundzir** [chakra-ui/chakra-ui#5704] [Q&A]
Hello, is there a way to style this selected label? I try to custom it but the things that can get customed is the 'control' and 'label' inside. What I want to achieve is to make the middle radio to have border on it's left and right side, thanks

![image](https://user-images.githubusercontent.com/51112444/157373212-ac20d0f7-0415-47e8-8da4-c5f59304852f.png)

**Comment by laoliveir** [comment]
Hi @muinmundzir , use parts: ['container'] to access the selected label
